css/03-generic.css

```diff
@@ -220,4 +220,28 @@ h4 {
 /* remove space between blog quotes and featured image */
 .entry-content .wp-block-pullquote:first-child {
 	margin-top: 0;
+}
+
+/* Blockquotes */
+
+blockquote.wp-block-quote {
+	background: var( --color-brandRed );
```

Comment:
I mentioned this in a comment I believe, but I think the black text on this red background is not enough contrast to pass accessibility standards. You can check the values here: https://contrastchecker.com/

Also, use `background-color` instead of the `background` shorthand - the shorthand can cause an unintentional reset of other properties ([more here](https://css-tricks.com/accidental-css-resets/)).

Reply:
Right - I had a question (that I can't find now either) about how strict that is. Should I follow it to the letter? Cuz if so, it ends up being pretty contrasty and saturated.

Reply:
@slambert at least AA for normal text is good!